bundles/org.openhab.binding.flicbutton/README.md

```diff
@@ -0,0 +1,119 @@
+# openHAB Flic Button Binding 
+
+openHAB binding for using [Flic Buttons](https://flic.io/)
+with a [fliclib-linux-hci](https://github.com/50ButtonsEach/fliclib-linux-hci) bridge.
+
+Currently, although Flic Buttons are BLE devices, this binding only supports fliclib-linux-hci (flicd)
+as a bridge. The openHAB Bluetooth Bindings are not supported. Flicd requires a seperate Bluetooth adapter to work, so
+if you use this binding together with e.g. the [Bluez Binding](https://www.openhab.org/addons/bindings/bluetooth.bluez/),
+two physical Bluetooth adapters are required (one for Bluez and one for flicd). Be aware that flicd requires an initial
+internet connection for the verification of the buttons. After buttons are initially added to flicd, internet connection
+is not required anymore.
```

Comment:
```suggestion
Currently, although Flic Buttons are BLE devices, this binding only supports fliclib-linux-hci (flicd)
as a bridge.
The openHAB Bluetooth Bindings are not supported.
Flicd requires a seperate Bluetooth adapter to work, so if you use this binding together with e.g. the [Bluez Binding](https://www.openhab.org/addons/bindings/bluetooth.bluez/), two physical Bluetooth adapters are required (one for Bluez and one for flicd). Be aware that flicd requires an initial internet connection for the verification of the buttons.
After buttons are initially added to flicd, internet connection is not required anymore.
```
All sentences should be on their own new line to reduce the changes should 1 line need to be edited after a merge.